2014/12/03 20:45:20 [error] 3653#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET /reports HTTP/1.1", upstream: "http://127.0.0.1:3000/reports", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/inspections"
2014/12/03 20:45:20 [error] 3653#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET /reports HTTP/1.1", upstream: "http://127.0.0.1:3001/reports", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/inspections"
2014/12/03 20:45:20 [error] 3653#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET /reports HTTP/1.1", upstream: "http://127.0.0.1:3002/reports", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/inspections"
2014/12/03 20:45:29 [error] 3653#0: *1 no live upstreams while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://dampers/", host: "dampers.cloudapp.net"
2014/12/03 20:47:18 [error] 3653#0: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 20:47:18 [error] 3653#0: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3001/", host: "dampers.cloudapp.net"
2014/12/03 20:47:18 [error] 3653#0: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "dampers.cloudapp.net"
2014/12/03 20:48:51 [error] 3653#0: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 20:48:51 [error] 3653#0: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "dampers.cloudapp.net"
2014/12/03 20:48:51 [error] 3653#0: *9 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3001/", host: "dampers.cloudapp.net"
2014/12/03 20:48:52 [error] 3653#0: *9 no live upstreams while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://dampers/", host: "dampers.cloudapp.net"
2014/12/03 20:50:44 [error] 3653#0: *13 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 20:50:44 [error] 3653#0: *13 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3001/", host: "dampers.cloudapp.net"
2014/12/03 20:50:44 [error] 3653#0: *13 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "dampers.cloudapp.net"
2014/12/03 20:51:20 [error] 3961#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "dampers.cloudapp.net"
2014/12/03 20:51:20 [error] 3961#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3001/", host: "dampers.cloudapp.net"
2014/12/03 20:51:20 [error] 3961#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 20:52:03 [error] 3961#0: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 125.64.35.67, server: cloudapp.net, request: "GET http://6.url.cn/zc/chs/img/body.png HTTP/1.1", upstream: "http://127.0.0.1:3002/zc/chs/img/body.png", host: "6.url.cn"
2014/12/03 20:52:03 [error] 3961#0: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 125.64.35.67, server: cloudapp.net, request: "GET http://6.url.cn/zc/chs/img/body.png HTTP/1.1", upstream: "http://127.0.0.1:3001/zc/chs/img/body.png", host: "6.url.cn"
2014/12/03 20:52:03 [error] 3961#0: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 125.64.35.67, server: cloudapp.net, request: "GET http://6.url.cn/zc/chs/img/body.png HTTP/1.1", upstream: "http://127.0.0.1:3000/zc/chs/img/body.png", host: "6.url.cn"
2014/12/03 20:53:33 [error] 4098#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "dampers.cloudapp.net"
2014/12/03 20:53:33 [error] 4098#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3001/", host: "dampers.cloudapp.net"
2014/12/03 20:53:33 [error] 4098#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 20:54:55 [error] 4183#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "dampers.cloudapp.net"
2014/12/03 20:54:55 [error] 4183#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3001/", host: "dampers.cloudapp.net"
2014/12/03 20:54:55 [error] 4183#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 20:56:31 [error] 4183#0: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 20:56:31 [error] 4183#0: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3001/", host: "dampers.cloudapp.net"
2014/12/03 20:56:31 [error] 4183#0: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "dampers.cloudapp.net"
2014/12/03 20:59:52 [error] 4326#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "dampers.cloudapp.net"
2014/12/03 20:59:52 [error] 4326#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3001/", host: "dampers.cloudapp.net"
2014/12/03 20:59:52 [error] 4326#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 21:06:40 [error] 4410#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "dampers.cloudapp.net"
2014/12/03 21:06:40 [error] 4410#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3001/", host: "dampers.cloudapp.net"
2014/12/03 21:06:40 [error] 4410#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 21:06:41 [error] 4410#0: *1 no live upstreams while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://dampers/", host: "dampers.cloudapp.net"
2014/12/03 21:06:42 [error] 4410#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 21:06:42 [error] 4410#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3001/", host: "dampers.cloudapp.net"
2014/12/03 21:06:42 [error] 4410#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "dampers.cloudapp.net"
2014/12/03 21:06:42 [error] 4410#0: *1 no live upstreams while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://dampers/", host: "dampers.cloudapp.net"
2014/12/03 21:06:42 [error] 4410#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 21:06:42 [error] 4410#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "dampers.cloudapp.net"
2014/12/03 21:06:42 [error] 4410#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3001/", host: "dampers.cloudapp.net"
2014/12/03 21:06:42 [error] 4410#0: *1 no live upstreams while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://dampers/", host: "dampers.cloudapp.net"
2014/12/03 21:06:42 [error] 4410#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 21:06:42 [error] 4410#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3001/", host: "dampers.cloudapp.net"
2014/12/03 21:06:42 [error] 4410#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "dampers.cloudapp.net"
2014/12/03 21:06:43 [error] 4410#0: *1 no live upstreams while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://dampers/", host: "dampers.cloudapp.net"
2014/12/03 21:06:43 [error] 4410#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 21:06:43 [error] 4410#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "dampers.cloudapp.net"
2014/12/03 21:06:43 [error] 4410#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3001/", host: "dampers.cloudapp.net"
2014/12/03 21:06:44 [error] 4410#0: *1 no live upstreams while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://dampers/", host: "dampers.cloudapp.net"
2014/12/03 21:08:52 [error] 4459#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "dampers.cloudapp.net"
2014/12/03 21:08:52 [error] 4459#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3001/", host: "dampers.cloudapp.net"
2014/12/03 21:08:52 [error] 4459#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 21:42:14 [error] 5255#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "dampers.cloudapp.net"
2014/12/03 21:42:14 [error] 5255#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3001/", host: "dampers.cloudapp.net"
2014/12/03 21:42:14 [error] 5255#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 21:43:27 [error] 5255#0: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 21:43:27 [error] 5255#0: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3001/", host: "dampers.cloudapp.net"
2014/12/03 21:43:40 [error] 5255#0: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 21:43:40 [error] 5255#0: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "dampers.cloudapp.net"
2014/12/03 21:43:40 [error] 5255#0: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3001/", host: "dampers.cloudapp.net"
2014/12/03 22:13:52 [error] 5255#0: *19 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 22:13:52 [error] 5255#0: *19 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3001/", host: "dampers.cloudapp.net"
2014/12/03 22:13:52 [error] 5255#0: *19 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "dampers.cloudapp.net"
2014/12/03 22:19:08 [error] 5255#0: *23 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 22:19:08 [error] 5255#0: *23 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "dampers.cloudapp.net"
2014/12/03 22:19:08 [error] 5255#0: *23 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3001/", host: "dampers.cloudapp.net"
2014/12/03 22:26:26 [error] 5255#0: *27 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 22:26:26 [error] 5255#0: *27 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3001/", host: "dampers.cloudapp.net"
2014/12/03 22:26:26 [error] 5255#0: *27 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "dampers.cloudapp.net"
2014/12/03 22:27:54 [error] 5255#0: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 22:27:54 [error] 5255#0: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "dampers.cloudapp.net"
2014/12/03 22:27:54 [error] 5255#0: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3001/", host: "dampers.cloudapp.net"
2014/12/03 22:27:55 [error] 5255#0: *31 no live upstreams while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://dampers/", host: "dampers.cloudapp.net"
2014/12/03 22:27:55 [error] 5255#0: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 22:27:55 [error] 5255#0: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3001/", host: "dampers.cloudapp.net"
2014/12/03 22:27:55 [error] 5255#0: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "dampers.cloudapp.net"
2014/12/03 22:27:56 [error] 5255#0: *31 no live upstreams while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://dampers/", host: "dampers.cloudapp.net"
2014/12/03 22:27:57 [error] 5255#0: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 22:27:57 [error] 5255#0: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "dampers.cloudapp.net"
2014/12/03 22:27:57 [error] 5255#0: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3001/", host: "dampers.cloudapp.net"
2014/12/03 22:27:57 [error] 5255#0: *31 no live upstreams while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://dampers/", host: "dampers.cloudapp.net"
2014/12/03 22:27:59 [error] 5255#0: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 22:27:59 [error] 5255#0: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3001/", host: "dampers.cloudapp.net"
2014/12/03 22:27:59 [error] 5255#0: *31 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "dampers.cloudapp.net"
2014/12/03 22:29:41 [error] 26153#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "dampers.cloudapp.net"
2014/12/03 22:29:41 [error] 26153#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3001/", host: "dampers.cloudapp.net"
2014/12/03 22:29:41 [error] 26153#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 22:29:43 [error] 26153#0: *1 no live upstreams while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://dampers/", host: "dampers.cloudapp.net"
2014/12/03 22:30:34 [error] 26153#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 22:30:34 [error] 26153#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3001/", host: "dampers.cloudapp.net"
2014/12/03 22:30:51 [error] 26153#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 22:30:51 [error] 26153#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "dampers.cloudapp.net"
2014/12/03 22:30:51 [error] 26153#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3001/", host: "dampers.cloudapp.net"
2014/12/03 22:31:19 [error] 26153#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 22:31:19 [error] 26153#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3001/", host: "dampers.cloudapp.net"
2014/12/03 22:31:19 [error] 26153#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3000/", host: "dampers.cloudapp.net"
2014/12/03 22:31:26 [error] 26153#0: *1 no live upstreams while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://dampers/", host: "dampers.cloudapp.net"
2014/12/03 22:31:27 [error] 26153#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET / HTTP/1.1", upstream: "http://127.0.0.1:3002/", host: "dampers.cloudapp.net"
2014/12/03 22:31:28 [error] 26153#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET /assets/application-41ec54657098a647ec6dc724d470e513.css HTTP/1.1", upstream: "http://127.0.0.1:3001/assets/application-41ec54657098a647ec6dc724d470e513.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/"
2014/12/03 22:31:44 [error] 26153#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET /assets/application-41ec54657098a647ec6dc724d470e513.css HTTP/1.1", upstream: "http://127.0.0.1:3002/assets/application-41ec54657098a647ec6dc724d470e513.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/"
2014/12/03 22:31:44 [error] 26153#0: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET /assets/application-41ec54657098a647ec6dc724d470e513.css HTTP/1.1", upstream: "http://127.0.0.1:3001/assets/application-41ec54657098a647ec6dc724d470e513.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/"
2014/12/03 22:32:02 [error] 26153#0: *24 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET /assets/application-fb5117133299f25b3f7395ce3177b5de.js HTTP/1.1", upstream: "http://127.0.0.1:3001/assets/application-fb5117133299f25b3f7395ce3177b5de.js", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/"
2014/12/03 22:32:02 [error] 26153#0: *24 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET /assets/application-fb5117133299f25b3f7395ce3177b5de.js HTTP/1.1", upstream: "http://127.0.0.1:3002/assets/application-fb5117133299f25b3f7395ce3177b5de.js", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/"
2014/12/03 22:49:03 [error] 26153#0: *56 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET /assets/application-41ec54657098a647ec6dc724d470e513.css HTTP/1.1", upstream: "http://127.0.0.1:3002/assets/application-41ec54657098a647ec6dc724d470e513.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/"
2014/12/03 22:49:03 [error] 26153#0: *56 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET /assets/application-41ec54657098a647ec6dc724d470e513.css HTTP/1.1", upstream: "http://127.0.0.1:3001/assets/application-41ec54657098a647ec6dc724d470e513.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/"
2014/12/03 22:49:41 [error] 26153#0: *56 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET /assets/application-41ec54657098a647ec6dc724d470e513.css HTTP/1.1", upstream: "http://127.0.0.1:3001/assets/application-41ec54657098a647ec6dc724d470e513.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/"
2014/12/03 22:49:41 [error] 26153#0: *62 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET /assets/application-fb5117133299f25b3f7395ce3177b5de.js HTTP/1.1", upstream: "http://127.0.0.1:3002/assets/application-fb5117133299f25b3f7395ce3177b5de.js", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/"
2014/12/03 22:49:55 [error] 26153#0: *56 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET /assets/application-41ec54657098a647ec6dc724d470e513.css HTTP/1.1", upstream: "http://127.0.0.1:3002/assets/application-41ec54657098a647ec6dc724d470e513.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/"
2014/12/03 22:49:55 [error] 26153#0: *56 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET /assets/application-41ec54657098a647ec6dc724d470e513.css HTTP/1.1", upstream: "http://127.0.0.1:3001/assets/application-41ec54657098a647ec6dc724d470e513.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/"
2014/12/03 22:50:33 [error] 26153#0: *56 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET /assets/application-41ec54657098a647ec6dc724d470e513.css HTTP/1.1", upstream: "http://127.0.0.1:3001/assets/application-41ec54657098a647ec6dc724d470e513.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/"
2014/12/03 22:50:33 [error] 26153#0: *62 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET /assets/application-fb5117133299f25b3f7395ce3177b5de.js HTTP/1.1", upstream: "http://127.0.0.1:3002/assets/application-fb5117133299f25b3f7395ce3177b5de.js", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/"
2014/12/03 22:51:03 [error] 26153#0: *56 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET /assets/application-41ec54657098a647ec6dc724d470e513.css HTTP/1.1", upstream: "http://127.0.0.1:3002/assets/application-41ec54657098a647ec6dc724d470e513.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/"
2014/12/03 22:51:03 [error] 26153#0: *56 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET /assets/application-41ec54657098a647ec6dc724d470e513.css HTTP/1.1", upstream: "http://127.0.0.1:3001/assets/application-41ec54657098a647ec6dc724d470e513.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/"
2014/12/03 22:51:18 [error] 26153#0: *56 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET /technicians HTTP/1.1", upstream: "http://127.0.0.1:3001/technicians", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/inspections"
2014/12/03 22:51:18 [error] 26153#0: *56 connect() failed (111: Connection refused) while connecting to upstream, client: 66.165.176.62, server: dampers.cloudapp.net, request: "GET /technicians HTTP/1.1", upstream: "http://127.0.0.1:3002/technicians", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/inspections"
2014/12/04 01:31:00 [error] 26153#0: *134 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "POST /login HTTP/1.1", upstream: "http://127.0.0.1:3002/login", host: "dampers.cloudapp.net"
2014/12/04 01:31:00 [error] 26153#0: *134 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "POST /login HTTP/1.1", upstream: "http://127.0.0.1:3001/login", host: "dampers.cloudapp.net"
2014/12/04 01:31:20 [error] 26153#0: *134 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 01:31:20 [error] 26153#0: *134 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 01:32:58 [error] 26153#0: *149 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 01:32:58 [error] 26153#0: *149 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 01:34:34 [error] 26153#0: *157 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "POST /inspections.json HTTP/1.1", upstream: "http://127.0.0.1:3001/inspections.json", host: "dampers.cloudapp.net"
2014/12/04 01:34:34 [error] 26153#0: *157 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "POST /inspections.json HTTP/1.1", upstream: "http://127.0.0.1:3002/inspections.json", host: "dampers.cloudapp.net"
2014/12/04 01:34:55 [error] 26153#0: *162 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-41ec54657098a647ec6dc724d470e513.css HTTP/1.1", upstream: "http://127.0.0.1:3002/assets/application-41ec54657098a647ec6dc724d470e513.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/"
2014/12/04 01:34:55 [error] 26153#0: *162 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-41ec54657098a647ec6dc724d470e513.css HTTP/1.1", upstream: "http://127.0.0.1:3001/assets/application-41ec54657098a647ec6dc724d470e513.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/"
2014/12/04 01:37:46 [error] 26153#0: *176 client intended to send too large body: 1588048 bytes, client: 98.242.253.39, server: dampers.cloudapp.net, request: "POST /inspections.json HTTP/1.1", host: "dampers.cloudapp.net"
2014/12/04 01:38:02 [error] 26153#0: *177 client intended to send too large body: 1588048 bytes, client: 98.242.253.39, server: dampers.cloudapp.net, request: "POST /inspections.json HTTP/1.1", host: "dampers.cloudapp.net"
2014/12/04 01:41:23 [error] 26153#0: *178 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 01:41:23 [error] 26153#0: *178 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 01:42:48 [error] 26153#0: *183 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "PUT /inspections/2.json HTTP/1.1", upstream: "http://127.0.0.1:3002/inspections/2.json", host: "dampers.cloudapp.net"
2014/12/04 01:42:48 [error] 26153#0: *183 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "PUT /inspections/2.json HTTP/1.1", upstream: "http://127.0.0.1:3001/inspections/2.json", host: "dampers.cloudapp.net"
2014/12/04 01:53:00 [error] 26153#0: *194 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 01:53:00 [error] 26153#0: *194 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 01:57:22 [error] 26153#0: *202 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 01:57:22 [error] 26153#0: *202 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 02:04:58 [error] 26153#0: *209 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 02:04:58 [error] 26153#0: *209 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 02:05:43 [error] 26153#0: *215 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /inspections/3 HTTP/1.1", upstream: "http://127.0.0.1:3002/inspections/3", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/inspections"
2014/12/04 02:05:43 [error] 26153#0: *215 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /inspections/3 HTTP/1.1", upstream: "http://127.0.0.1:3001/inspections/3", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/inspections"
2014/12/04 02:05:57 [error] 26153#0: *215 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /inspections/2 HTTP/1.1", upstream: "http://127.0.0.1:3001/inspections/2", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/inspections"
2014/12/04 02:05:57 [error] 26153#0: *215 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /inspections/2 HTTP/1.1", upstream: "http://127.0.0.1:3002/inspections/2", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/inspections"
2014/12/04 02:06:12 [error] 26153#0: *215 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /inspections HTTP/1.1", upstream: "http://127.0.0.1:3002/inspections", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/inspections/3"
2014/12/04 02:06:12 [error] 26153#0: *215 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /inspections HTTP/1.1", upstream: "http://127.0.0.1:3001/inspections", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/inspections/3"
2014/12/04 02:09:22 [error] 26153#0: *251 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-1e6a99961e90e3292a16a52fb39b4781.js HTTP/1.1", upstream: "http://127.0.0.1:3001/assets/application-1e6a99961e90e3292a16a52fb39b4781.js", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 02:09:22 [error] 26153#0: *251 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-1e6a99961e90e3292a16a52fb39b4781.js HTTP/1.1", upstream: "http://127.0.0.1:3002/assets/application-1e6a99961e90e3292a16a52fb39b4781.js", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 02:13:22 [error] 26153#0: *266 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 02:13:22 [error] 26153#0: *266 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 02:18:06 [error] 26153#0: *274 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 02:18:06 [error] 26153#0: *274 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 02:20:34 [error] 26153#0: *282 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 02:20:34 [error] 26153#0: *282 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 02:21:34 [error] 26153#0: *288 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /inspections/job/1.json HTTP/1.1", upstream: "http://127.0.0.1:3001/inspections/job/1.json", host: "dampers.cloudapp.net"
2014/12/04 02:21:34 [error] 26153#0: *288 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /inspections/job/1.json HTTP/1.1", upstream: "http://127.0.0.1:3002/inspections/job/1.json", host: "dampers.cloudapp.net"
2014/12/04 02:22:41 [error] 26153#0: *297 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 02:22:41 [error] 26153#0: *297 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 02:27:50 [error] 26153#0: *304 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 02:27:50 [error] 26153#0: *304 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 02:28:49 [error] 26153#0: *311 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 02:28:49 [error] 26153#0: *311 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 02:30:46 [error] 26153#0: *319 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 02:30:46 [error] 26153#0: *319 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 02:35:25 [error] 26153#0: *326 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /inspections HTTP/1.1", upstream: "http://127.0.0.1:3002/inspections", host: "dampers.cloudapp.net"
2014/12/04 02:35:25 [error] 26153#0: *327 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /technicians HTTP/1.1", upstream: "http://127.0.0.1:3001/technicians", host: "dampers.cloudapp.net"
2014/12/04 02:37:41 [error] 26153#0: *345 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 02:37:41 [error] 26153#0: *345 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 02:39:05 [error] 26153#0: *353 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /inspections HTTP/1.1", upstream: "http://127.0.0.1:3002/inspections", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/inspections"
2014/12/04 02:39:05 [error] 26153#0: *353 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /inspections HTTP/1.1", upstream: "http://127.0.0.1:3001/inspections", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/inspections"
2014/12/04 02:42:22 [error] 26153#0: *364 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 02:42:22 [error] 26153#0: *364 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 02:44:06 [error] 26153#0: *372 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 02:44:06 [error] 26153#0: *372 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 02:49:15 [error] 26153#0: *380 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 02:49:15 [error] 26153#0: *380 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 03:00:25 [error] 26153#0: *388 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 03:00:25 [error] 26153#0: *388 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 03:09:16 [error] 26153#0: *396 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 03:09:16 [error] 26153#0: *396 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 03:12:03 [error] 26153#0: *401 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "PUT /inspections/3.json HTTP/1.1", upstream: "http://127.0.0.1:3002/inspections/3.json", host: "dampers.cloudapp.net"
2014/12/04 03:12:03 [error] 26153#0: *401 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "PUT /inspections/3.json HTTP/1.1", upstream: "http://127.0.0.1:3001/inspections/3.json", host: "dampers.cloudapp.net"
2014/12/04 03:12:25 [error] 26153#0: *410 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-1e6a99961e90e3292a16a52fb39b4781.js HTTP/1.1", upstream: "http://127.0.0.1:3001/assets/application-1e6a99961e90e3292a16a52fb39b4781.js", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 03:12:25 [error] 26153#0: *410 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-1e6a99961e90e3292a16a52fb39b4781.js HTTP/1.1", upstream: "http://127.0.0.1:3002/assets/application-1e6a99961e90e3292a16a52fb39b4781.js", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 03:14:19 [error] 26153#0: *414 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "POST /inspections.json HTTP/1.1", upstream: "http://127.0.0.1:3002/inspections.json", host: "dampers.cloudapp.net"
2014/12/04 03:14:19 [error] 26153#0: *414 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "POST /inspections.json HTTP/1.1", upstream: "http://127.0.0.1:3001/inspections.json", host: "dampers.cloudapp.net"
2014/12/04 03:14:46 [error] 26153#0: *422 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /inspections HTTP/1.1", upstream: "http://127.0.0.1:3001/inspections", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/inspections"
2014/12/04 03:14:46 [error] 26153#0: *422 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /inspections HTTP/1.1", upstream: "http://127.0.0.1:3002/inspections", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/inspections"
2014/12/04 03:15:51 [error] 26153#0: *429 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "POST /inspections.json HTTP/1.1", upstream: "http://127.0.0.1:3002/inspections.json", host: "dampers.cloudapp.net"
2014/12/04 03:15:51 [error] 26153#0: *429 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "POST /inspections.json HTTP/1.1", upstream: "http://127.0.0.1:3001/inspections.json", host: "dampers.cloudapp.net"
2014/12/04 03:16:08 [error] 26153#0: *429 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 03:16:08 [error] 26153#0: *429 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 03:16:21 [error] 26153#0: *439 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-1e6a99961e90e3292a16a52fb39b4781.js HTTP/1.1", upstream: "http://127.0.0.1:3002/assets/application-1e6a99961e90e3292a16a52fb39b4781.js", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 03:16:21 [error] 26153#0: *439 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-1e6a99961e90e3292a16a52fb39b4781.js HTTP/1.1", upstream: "http://127.0.0.1:3001/assets/application-1e6a99961e90e3292a16a52fb39b4781.js", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 03:17:43 [error] 26153#0: *456 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /inspections/job/1.json HTTP/1.1", upstream: "http://127.0.0.1:3001/inspections/job/1.json", host: "dampers.cloudapp.net"
2014/12/04 03:17:43 [error] 26153#0: *456 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /inspections/job/1.json HTTP/1.1", upstream: "http://127.0.0.1:3002/inspections/job/1.json", host: "dampers.cloudapp.net"
2014/12/04 03:20:39 [error] 26153#0: *465 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-1e6a99961e90e3292a16a52fb39b4781.js HTTP/1.1", upstream: "http://127.0.0.1:3002/assets/application-1e6a99961e90e3292a16a52fb39b4781.js", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 03:20:39 [error] 26153#0: *465 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-1e6a99961e90e3292a16a52fb39b4781.js HTTP/1.1", upstream: "http://127.0.0.1:3001/assets/application-1e6a99961e90e3292a16a52fb39b4781.js", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 03:21:00 [error] 26153#0: *462 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-0d0b300461a603a719c591568d28def6.css HTTP/1.1", upstream: "http://127.0.0.1:3001/assets/application-0d0b300461a603a719c591568d28def6.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 03:21:00 [error] 26153#0: *462 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-0d0b300461a603a719c591568d28def6.css HTTP/1.1", upstream: "http://127.0.0.1:3002/assets/application-0d0b300461a603a719c591568d28def6.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 03:22:15 [error] 26153#0: *481 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 03:22:15 [error] 26153#0: *481 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 03:22:28 [error] 26153#0: *489 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-1e6a99961e90e3292a16a52fb39b4781.js HTTP/1.1", upstream: "http://127.0.0.1:3001/assets/application-1e6a99961e90e3292a16a52fb39b4781.js", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 03:22:28 [error] 26153#0: *489 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-1e6a99961e90e3292a16a52fb39b4781.js HTTP/1.1", upstream: "http://127.0.0.1:3002/assets/application-1e6a99961e90e3292a16a52fb39b4781.js", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 03:22:44 [error] 26153#0: *489 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-1e6a99961e90e3292a16a52fb39b4781.js HTTP/1.1", upstream: "http://127.0.0.1:3002/assets/application-1e6a99961e90e3292a16a52fb39b4781.js", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 03:22:44 [error] 26153#0: *489 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-1e6a99961e90e3292a16a52fb39b4781.js HTTP/1.1", upstream: "http://127.0.0.1:3001/assets/application-1e6a99961e90e3292a16a52fb39b4781.js", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 03:27:01 [error] 26153#0: *504 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 03:27:01 [error] 26153#0: *504 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 03:27:57 [error] 26153#0: *510 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /technicians/current HTTP/1.1", upstream: "http://127.0.0.1:3002/technicians/current", host: "dampers.cloudapp.net"
2014/12/04 03:27:57 [error] 26153#0: *510 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /technicians/current HTTP/1.1", upstream: "http://127.0.0.1:3001/technicians/current", host: "dampers.cloudapp.net"
2014/12/04 03:30:24 [error] 26153#0: *519 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 03:30:24 [error] 26153#0: *519 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 03:30:36 [error] 26153#0: *529 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-1e6a99961e90e3292a16a52fb39b4781.js HTTP/1.1", upstream: "http://127.0.0.1:3002/assets/application-1e6a99961e90e3292a16a52fb39b4781.js", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 03:30:36 [error] 26153#0: *529 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-1e6a99961e90e3292a16a52fb39b4781.js HTTP/1.1", upstream: "http://127.0.0.1:3001/assets/application-1e6a99961e90e3292a16a52fb39b4781.js", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 03:31:32 [error] 26153#0: *533 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "PUT /inspections/9.json HTTP/1.1", upstream: "http://127.0.0.1:3001/inspections/9.json", host: "dampers.cloudapp.net"
2014/12/04 03:31:32 [error] 26153#0: *533 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "PUT /inspections/9.json HTTP/1.1", upstream: "http://127.0.0.1:3002/inspections/9.json", host: "dampers.cloudapp.net"
2014/12/04 03:33:13 [error] 26153#0: *540 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "PUT /inspections/9.json HTTP/1.1", upstream: "http://127.0.0.1:3002/inspections/9.json", host: "dampers.cloudapp.net"
2014/12/04 03:33:13 [error] 26153#0: *540 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "PUT /inspections/9.json HTTP/1.1", upstream: "http://127.0.0.1:3001/inspections/9.json", host: "dampers.cloudapp.net"
2014/12/04 03:38:49 [error] 26153#0: *555 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 03:38:49 [error] 26153#0: *555 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 03:40:43 [error] 26153#0: *563 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 03:40:43 [error] 26153#0: *563 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 03:42:06 [error] 26153#0: *571 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 03:42:06 [error] 26153#0: *571 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 03:43:53 [error] 26153#0: *579 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 03:43:53 [error] 26153#0: *579 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 03:47:50 [error] 26153#0: *587 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 03:47:50 [error] 26153#0: *587 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 03:48:39 [error] 26153#0: *592 client intended to send too large body: 1065512 bytes, client: 98.242.253.39, server: dampers.cloudapp.net, request: "PUT /inspections/9.json HTTP/1.1", host: "dampers.cloudapp.net"
2014/12/04 03:49:20 [error] 26153#0: *594 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /inspections/job/1.json HTTP/1.1", upstream: "http://127.0.0.1:3002/inspections/job/1.json", host: "dampers.cloudapp.net"
2014/12/04 03:49:20 [error] 26153#0: *594 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /inspections/job/1.json HTTP/1.1", upstream: "http://127.0.0.1:3001/inspections/job/1.json", host: "dampers.cloudapp.net"
2014/12/04 03:50:36 [error] 26153#0: *601 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "POST /inspections.json HTTP/1.1", upstream: "http://127.0.0.1:3001/inspections.json", host: "dampers.cloudapp.net"
2014/12/04 03:50:36 [error] 26153#0: *601 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "POST /inspections.json HTTP/1.1", upstream: "http://127.0.0.1:3002/inspections.json", host: "dampers.cloudapp.net"
2014/12/04 03:51:07 [error] 26153#0: *607 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-0d0b300461a603a719c591568d28def6.css HTTP/1.1", upstream: "http://127.0.0.1:3002/assets/application-0d0b300461a603a719c591568d28def6.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 03:51:07 [error] 26153#0: *607 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-0d0b300461a603a719c591568d28def6.css HTTP/1.1", upstream: "http://127.0.0.1:3001/assets/application-0d0b300461a603a719c591568d28def6.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 03:52:53 [error] 26153#0: *618 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-0d0b300461a603a719c591568d28def6.css HTTP/1.1", upstream: "http://127.0.0.1:3001/assets/application-0d0b300461a603a719c591568d28def6.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 03:52:53 [error] 26153#0: *618 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-0d0b300461a603a719c591568d28def6.css HTTP/1.1", upstream: "http://127.0.0.1:3002/assets/application-0d0b300461a603a719c591568d28def6.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 03:54:24 [error] 26153#0: *630 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /technicians/current HTTP/1.1", upstream: "http://127.0.0.1:3002/technicians/current", host: "dampers.cloudapp.net"
2014/12/04 03:54:24 [error] 26153#0: *630 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /technicians/current HTTP/1.1", upstream: "http://127.0.0.1:3001/technicians/current", host: "dampers.cloudapp.net"
2014/12/04 03:58:24 [error] 26153#0: *639 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3001/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 03:58:24 [error] 26153#0: *639 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /jobs.json HTTP/1.1", upstream: "http://127.0.0.1:3002/jobs.json", host: "dampers.cloudapp.net"
2014/12/04 04:06:54 [error] 26153#0: *645 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /logout HTTP/1.1", upstream: "http://127.0.0.1:3002/logout", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/reports"
2014/12/04 04:06:54 [error] 26153#0: *645 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /logout HTTP/1.1", upstream: "http://127.0.0.1:3001/logout", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/reports"
2014/12/04 04:07:06 [error] 26153#0: *645 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-0d0b300461a603a719c591568d28def6.css HTTP/1.1", upstream: "http://127.0.0.1:3001/assets/application-0d0b300461a603a719c591568d28def6.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 04:07:06 [error] 26153#0: *645 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-0d0b300461a603a719c591568d28def6.css HTTP/1.1", upstream: "http://127.0.0.1:3002/assets/application-0d0b300461a603a719c591568d28def6.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 04:12:15 [error] 26153#0: *670 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /reports HTTP/1.1", upstream: "http://127.0.0.1:3002/reports", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/reports"
2014/12/04 04:12:15 [error] 26153#0: *670 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /reports HTTP/1.1", upstream: "http://127.0.0.1:3001/reports", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/reports"
2014/12/04 04:12:31 [error] 26153#0: *670 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-0d0b300461a603a719c591568d28def6.css HTTP/1.1", upstream: "http://127.0.0.1:3001/assets/application-0d0b300461a603a719c591568d28def6.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 04:12:31 [error] 26153#0: *670 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-0d0b300461a603a719c591568d28def6.css HTTP/1.1", upstream: "http://127.0.0.1:3002/assets/application-0d0b300461a603a719c591568d28def6.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 04:13:18 [error] 26153#0: *670 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /inspections HTTP/1.1", upstream: "http://127.0.0.1:3002/inspections", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 04:13:18 [error] 26153#0: *670 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /inspections HTTP/1.1", upstream: "http://127.0.0.1:3001/inspections", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 04:17:45 [error] 26153#0: *698 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /reports HTTP/1.1", upstream: "http://127.0.0.1:3001/reports", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/reports/new"
2014/12/04 04:17:45 [error] 26153#0: *698 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /reports HTTP/1.1", upstream: "http://127.0.0.1:3002/reports", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/reports/new"
2014/12/04 04:28:37 [error] 26153#0: *707 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /reports HTTP/1.1", upstream: "http://127.0.0.1:3002/reports", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/reports/new"
2014/12/04 04:28:37 [error] 26153#0: *707 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /reports HTTP/1.1", upstream: "http://127.0.0.1:3001/reports", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/reports/new"
2014/12/04 04:30:40 [error] 26153#0: *718 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /reports HTTP/1.1", upstream: "http://127.0.0.1:3001/reports", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/reports"
2014/12/04 04:30:40 [error] 26153#0: *718 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /reports HTTP/1.1", upstream: "http://127.0.0.1:3002/reports", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/reports"
2014/12/04 04:30:51 [error] 26153#0: *718 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-0d0b300461a603a719c591568d28def6.css HTTP/1.1", upstream: "http://127.0.0.1:3002/assets/application-0d0b300461a603a719c591568d28def6.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 04:30:51 [error] 26153#0: *718 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-0d0b300461a603a719c591568d28def6.css HTTP/1.1", upstream: "http://127.0.0.1:3001/assets/application-0d0b300461a603a719c591568d28def6.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 04:33:54 [error] 26153#0: *741 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "POST /reports HTTP/1.1", upstream: "http://127.0.0.1:3001/reports", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/reports/new"
2014/12/04 04:33:54 [error] 26153#0: *741 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "POST /reports HTTP/1.1", upstream: "http://127.0.0.1:3002/reports", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/reports/new"
2014/12/04 04:34:39 [error] 26153#0: *741 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /reports/6.pdf HTTP/1.1", upstream: "http://127.0.0.1:3002/reports/6.pdf", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/reports"
2014/12/04 04:34:39 [error] 26153#0: *741 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /reports/6.pdf HTTP/1.1", upstream: "http://127.0.0.1:3001/reports/6.pdf", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/reports"
2014/12/04 04:36:25 [error] 26153#0: *757 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /reports HTTP/1.1", upstream: "http://127.0.0.1:3001/reports", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/reports/new"
2014/12/04 04:36:25 [error] 26153#0: *757 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /reports HTTP/1.1", upstream: "http://127.0.0.1:3002/reports", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/reports/new"
2014/12/04 04:39:31 [error] 26153#0: *767 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /reports HTTP/1.1", upstream: "http://127.0.0.1:3002/reports", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/reports/new"
2014/12/04 04:39:31 [error] 26153#0: *767 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /reports HTTP/1.1", upstream: "http://127.0.0.1:3001/reports", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/reports/new"
2014/12/04 04:41:24 [error] 26153#0: *777 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /reports HTTP/1.1", upstream: "http://127.0.0.1:3001/reports", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/reports/new"
2014/12/04 04:41:24 [error] 26153#0: *777 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /reports HTTP/1.1", upstream: "http://127.0.0.1:3002/reports", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/reports/new"
2014/12/04 04:46:13 [error] 26153#0: *788 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /reports HTTP/1.1", upstream: "http://127.0.0.1:3002/reports", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/reports"
2014/12/04 04:46:13 [error] 26153#0: *788 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /reports HTTP/1.1", upstream: "http://127.0.0.1:3001/reports", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/reports"
2014/12/04 04:46:27 [error] 26153#0: *788 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-0d0b300461a603a719c591568d28def6.css HTTP/1.1", upstream: "http://127.0.0.1:3001/assets/application-0d0b300461a603a719c591568d28def6.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 04:46:27 [error] 26153#0: *788 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-0d0b300461a603a719c591568d28def6.css HTTP/1.1", upstream: "http://127.0.0.1:3002/assets/application-0d0b300461a603a719c591568d28def6.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 04:46:38 [error] 26153#0: *788 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-0d0b300461a603a719c591568d28def6.css HTTP/1.1", upstream: "http://127.0.0.1:3002/assets/application-0d0b300461a603a719c591568d28def6.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 04:46:38 [error] 26153#0: *788 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-0d0b300461a603a719c591568d28def6.css HTTP/1.1", upstream: "http://127.0.0.1:3001/assets/application-0d0b300461a603a719c591568d28def6.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 04:46:55 [error] 26153#0: *788 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /reports/10.pdf HTTP/1.1", upstream: "http://127.0.0.1:3001/reports/10.pdf", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/reports"
2014/12/04 04:46:55 [error] 26153#0: *788 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /reports/10.pdf HTTP/1.1", upstream: "http://127.0.0.1:3002/reports/10.pdf", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/reports"
2014/12/04 04:47:17 [error] 26153#0: *788 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-0d0b300461a603a719c591568d28def6.css HTTP/1.1", upstream: "http://127.0.0.1:3002/assets/application-0d0b300461a603a719c591568d28def6.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 04:47:17 [error] 26153#0: *788 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /assets/application-0d0b300461a603a719c591568d28def6.css HTTP/1.1", upstream: "http://127.0.0.1:3001/assets/application-0d0b300461a603a719c591568d28def6.css", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/technicians"
2014/12/04 04:48:45 [error] 26153#0: *831 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /reports HTTP/1.1", upstream: "http://127.0.0.1:3001/reports", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/reports/new"
2014/12/04 04:48:45 [error] 26153#0: *831 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "GET /reports HTTP/1.1", upstream: "http://127.0.0.1:3002/reports", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/reports/new"
2014/12/04 04:49:22 [error] 26153#0: *831 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "POST /reports/11 HTTP/1.1", upstream: "http://127.0.0.1:3002/reports/11", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/reports"
2014/12/04 04:49:22 [error] 26153#0: *831 connect() failed (111: Connection refused) while connecting to upstream, client: 98.242.253.39, server: dampers.cloudapp.net, request: "POST /reports/11 HTTP/1.1", upstream: "http://127.0.0.1:3001/reports/11", host: "dampers.cloudapp.net", referrer: "http://dampers.cloudapp.net/reports"
